data/src/main/java/pl/edu/agh/model/LocalDateTimePeriod.java

```diff
@@ -0,0 +1,66 @@
+package pl.edu.agh.model;
+
+import java.time.DateTimeException;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
+@AllArgsConstructor
+@Embeddable
+public class LocalDateTimePeriod {
+
+    @Column(nullable = false)
+    private LocalDateTime begin;
```

Comment:
Ja bym tu głosowała za `beginning` jednak, żeby rzeczownik był

Reply:
za `ending` tez bys glosowala? ;d

Reply:
Hmm, ja w sumie nie wiem jakoś super wiele o cpp? Generalnie `begin` jest czasownikiem, a `end` rzeczownikiem i czasownikiem. Pola w klasie mi bardziej pasują jako rzeczowniki jednak (początek i koniec), także `beginning` i `end` dla mnie brzmią najlepiej

Mogę bardziej pod wieczór zaangażować się w SZUKANIE GŁOSÓW EKSPERTÓW, ale na razie mam [losowy link z jakiegoś słownika](https://context.reverso.net/t%C5%82umaczenie/angielski-polski/from+beginning+to+end)

Reply:
mnie to razi w oczy, ale jak ktos jeszcze jest za to zmienie

Reply:
dam start, mysle ze to bedzie dobry kompromis

Reply:
@szbrnl @kjarosh @MrMarosis WAŻNE USTALENIA

Reply:
https://docs.oracle.com/javase/8/docs/api/java/time/Period.html#between-java.time.LocalDate-java.time.LocalDate-

tutaj mają startDate i endDate

Reply:
dajmy więc `startTime` i `endTime` oraz zakończmy te jakże ważne ustalenia